project/profiles/capacity4more/modules/c4m/content/c4m_content_document/c4m_content_document.module

```diff
@@ -24,6 +24,59 @@ function c4m_content_document_c4m_og_vocab_info_content() {
 }
 
 /**
+ * Author autocomplete callback.
+ */
+function _edit_author_autocomplete($group_type, $gid, $string) {
+  extract(og_context());
+  $query = db_select('users', 'u');
+  $query
+    ->condition('u.uid', 0, '<>')
+    ->condition('u.status', 1, '=')
+    ->condition('u.name', '%' . db_like($string) . '%', 'LIKE')
+    ->fields('u', array('uid', 'name'))
+    ->join('og_membership', 'ogm', "ogm.gid = :gid AND u.uid = ogm.etid AND ogm.entity_type = 'user'", array(':gid' => $gid));
+  $result = $query->execute();
+
+  foreach ($result as $member) {
+    $user[$member->name] = $member->name;
+  }
+
+  // Convert to object to prevent drupal bug, see http://drupal.org/node/175361
+  drupal_json_output((object) $user);
+}
+
+/**
+ * Implements hook_menu().
+ */
+function c4m_content_document_menu() {
+  $menu = array();
+
+  // Auto-completes available user names.
+  $menu['edit-author/autocomplete'] = array(
+    'page callback' => '_edit_author_autocomplete',
+    'page arguments' => array(1, 2),
+    'access callback' => '_c4m_features_og_members_is_power_user',
+    'access arguments' => array(1, 2),
```

Comment:
this menu item has only 2 indexes, here we try to send the third one which doesn't exists ...